### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by FloSchuberth** [jasp-stats/jasp-issues#2089]
### Description

Partial Least Squares SEM

### Purpose

Ensure proper implementation

### Is your feature request related to a JASP module?

SEM

### Additional context

I saw that you use cSEM to implement PLS-SEM in JASP. First of all thank you and I am very happy that you have chosen my package. 

However, I think there is still a lot room for improving the implementation. For example you currently offer a lot of features, which are hardly relevant for users, e.g., the weighting approach. 
Moreover, there are mistakes in the interface, e.g., Error calculation method including the options provided does not make sense. Further, some important features such as the overall model fit test and other commonly reported statistics, e.g., AVE or HTMT are not provided.

I am happy to help improving your current implementation. Perhaps the person who is responsible for PLS-SEM wants to get in touch with me (f.schuberth@utwente.nl)

**Comment by Kucharssim** [comment]
@FloSchuberth, thank you for the feedback and offering your help! We appreciate it. I contacted you through e-mail to discuss the specific points you raised.

**Comment by francischuah** [comment]
Hello everyone, is there any update on this? I start using JASP PLS SEM package for classes and found that it is very helpful.

On this side of the world (far east), cSEM in R has a steep learning curve while paid software might not be affordable for graduate student here. So the initiative started by JASP team to include PLS is very much commendable.

The present output produce by PLS SEM JASP is appropriate for assessment of structural model and some aspect of measurement model.

I do hope that more measurement model assessment criteria can be included in line with the output generated by cSEM.

I hope it is not too much to ask as I know everything about JASP is an effort from the community to the community.

**Comment by juliuspfadt** [comment]
What is the current state of this @Kucharssim?

**Comment by tomtomme** [comment]
@FloSchuberth 
Could you work something out with Kucharssim? Are you working on improvements? Thx in advance!

**Comment by FloSchuberth** [comment]
@tomtomme: Yes sure, I can support the person who is responsible for the maintenance of the PLS-SEM JASP module. Not sure what you exactly mean by "Are you working on improvements". I am the authors of the cSEM package. I think the current issue is that a lot of cSEM features/output provided are currently not used in JASP. So I could let you know how to obtain the AVEs, VIF, HTMTs in cSEM...

**Comment by tomtomme** [comment]
@FloSchuberth 
Nice. I meant your email contact with Kucharssim, and if there is any progress regarding jasps pls sem module. It sounded like you might want to work on it directly. 
Current maintainer of pls sem in jasp is julius who is assigned to this issue. Lets cross fingers that he can find the time to work and this and contact you.
Thx so far.

**Comment by FloSchuberth** [comment]
@tomtomme: Some time ago (I guess over a year ago) I had contact with one member of your team (not sure whether it was Kucharssim). Then PLS-SEM was implemented in JASP and we did one round of improvements. Since then nothing happened. 
Considering working directly on the JASP implementation, could also be an option, however, I have not implemented things before in JASP, and thus would need to familiarize myself wit the JASPs structure, workflow, etc. 
In terms of efficiency, I think it would be easier to meet with julius and tell him how to find/produce the requested output in cSEM, so that he can implement in JASP. That having said, @juliuspfadt feel free to contact me (f.schuberth@utwente.nl) and we can arrange a meeting.

**Comment by tomtomme** [comment]
@FloSchuberth 
Thx again. 
If you feel you have time for some inefficiency, consider to jump right in and make a PR.

The dev workflow to create a module is detailed here:
https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-module-workflow.md

To upgrade an existing module see
https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-upgrade-qml.md

The team certainly can use every help it can get. Cheers :)

**Comment by FloSchuberth** [comment]
@tomtomme 
:) I will have a look at it, but of course can't promise anything.

**Comment by tomtomme** [comment]
now related to https://github.com/jasp-stats/jasp-issues/issues/2601
But not a duplicate since this issue wants 

- [ ] overall model fit test 
- [ ] AVE 
- [ ] VIF

Tracked at #2601 
- [x] HTMT
- [x] set PLS as default weight and nuke weight setting
- [x] Error calculation method +options do not make sense?
